source/content/guides/collaborative-development.md

```diff
@@ -8,379 +8,335 @@ permalink: docs/guides/:basename
+  github  git@github.com:pantheon-learning/d7-ci.git (push)
+  origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (fetch)
+  origin  ssh://codeserver.dev.UUID@codeserver.dev.UUID.drush.in:2222/~/repository.git (push)
+  ```
 
-### Push the Pantheon Site's Codebase to GitHub
+#### Configure Origin as a Multi-Remote Destination (Optional)
 
-```bash{outputLines: 2-6}
-git push -u github master
-Writing objects: 100% (120046/120046), 31.88 MiB | 3.26 MiB/s, done.
-Total 120046 (delta 89679), reused 120039 (delta 89679)
-To git@github.com:pantheon-learning/d7-ci.git
- * [new branch]      master -> master
-Branch master set up to track remote branch master from github.
-```
+  Another option is to add another push URL for origin.
 
-The repository on GitHub now has all of the same code as my site.
+1. Add a push URL within `.git/config`:
 
-![GitHub repo same as site](../../images/github-repo-origin-sameas-remote.png)
+  ```none:title=.git/config
+  [remote "origin"]
+    url = ssh://codeserver.dev.<SITE_UUID>@codeserver.dev.<SITE_UUID>.drush.in:2222/~/repository.git
+    url = git@github.com:systemseed/example.git
+  ```
 
-## Synchronizing Existing Site Repositories on GitHub With Pantheon
+  Commits will be pushed to both remote destinations automatically on `git push origin`. Enforce this configuration with all team members when working collaboratively. Props to [Tom Kirkpatrick](/contributors/mrfelton) for contributing this tip in the [Pantheon Community](/pantheon-community).
 
-This process will follow the same general procedures as moving your repo from Pantheon to GitHub, in reverse.
+### Push the Pantheon Site's Codebase to GitHub
 
-### Create the Site
+1. Enter the code below:
 
-At our [site creation page](https://dashboard.pantheon.io/sites/create/ "Pantheon's site creation page, used to select an upstream"), name your site. On the next page, `/sites/UUID/configure` select the upstream your site will track. If you need a Custom Upstream, like one managed by your organization, or one of the public upstreams running on the platform, it is important that you create it as a new site based on that upstream. This will set the upstream for your site, which cannot be changed.
+  ```bash{outputLines: 2-6}
+  git push -u github master
+  Writing objects: 100% (120046/120046), 31.88 MiB | 3.26 MiB/s, done.
+  Total 120046 (delta 89679), reused 120039 (delta 89679)
+  To git@github.com:pantheon-learning/d7-ci.git
+  ```
 
-### Pull in Pantheon's Upstream
+ The repository on GitHub now has all of the same code as the existing site.
 
-As long as you've chosen the same codebase (Drupal 7, WordPress, Commerce Kickstart, etc.) as the starting point of your Pantheon site, you can use Git to import your existing code with your commit history intact, while also preserving Pantheon's [upstream update](/core-updates) function.
+ ![GitHub repo same as site](../../images/github-repo-origin-sameas-remote.png)
```

Comment:
@whitneymeredith - This one is also blurry

Reply:
@michellecolon-pantheon image replaced